resources/reporters/currents-playwright/README.md

```
@@ -159,10 +184,17 @@ pwc --key <record-key> --project-id <id>
 * Run orchestration for all tests in the current directory:
 
 ```
-pwc-p --key <record-key> --project-id <id> --ci-build-id <build-id>
+pwc-p run --key <record-key> --project-id <id> --ci-build-id <build-id>
+```
+
+* Run orchestration for tests filtered by the tag "@smoke":
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)